content/README.md

```diff
@@ -1,11 +1,11 @@
 # Content Structure
 
 In this folder you will find content for the following:
-* meetings
-* talks
-* speakers
-* hosts
-* sponsors
+* [meetings](https://github.com/dsmjs/site/tree/master/content/meetings)
+* [talks](https://github.com/dsmjs/site/tree/master/content/talks)
+* [speakers](https://github.com/dsmjs/site/tree/master/content/speakers)
+* [hosts](https://github.com/dsmjs/site/tree/master/content/hosts)
+* [sponsors](https://github.com/dsmjs/site/tree/master/content/sponsors)
```

Comment:
i think these can be done as relative links. that has the benefit of working in other branches and even locally with some editors